public/components/freelancer.js

```diff
@@ -580,6 +582,134 @@ class Tags extends React.Component {
   }
 }
 
+class UserLink extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  redirectUserPage(userlink) {
+    return function(ev) {
+      ev.preventDefault();
+      ev.stopPropagation();
+      window.location = '/user/' + userlink.props.owner.username;
+    }
+  }
+
+  render() {
+
+    // there is an owner
+    if (this.props.owner) {
+      let who = " by user " + this.props.owner.username;
+      let loggedUserEl = document.getElementById('freelancer-logged-reviews-root');
+      console.log(loggedUserEl);
+      if (loggedUserEl) {
+        let loggedUserName = loggedUserEl.getAttribute('data-username');
+        if (loggedUserName == this.props.owner.username) {
+          console.log("HERE IT WORKS LOOK");
```

Comment:
I was angry at the code because it refused to work when I had written `if (loggedUserName = this.props.owner.username)`